src/eve/traits.py

```diff
@@ -68,3 +68,16 @@ def _collect_symbols_validator(  # type: ignore  # validators are classmethods
     def collect_symbols(self) -> None:
         self.symtable_ = dict()
         self.symtable_ = self._collect_symbols(self)
+
+    @staticmethod
+    def add_symtable(
+        node: concepts.Node, visitor: visitors.NodeVisitor, kwargs: Dict[str, Any]
+    ) -> None:
+        """Update or add the symtable to kwargs in the visitor calls.
+
+        This is a previsitor that is registered with classes in eve.visitors
+        by passing this to the base class constructor.
```

Comment:
This docstring should be updated. It refers to the previous implementation of the feature